packages/systemd-257/systemd-257.spec

```diff
@@ -0,0 +1,799 @@
+# Skip check-rpaths since we expect them for systemd.
+%global __brp_check_rpaths %{nil}
+
+%global debug_package %{nil}
```

Comment:
This is almost always an error and you should fix whatever is causing the debuginfo package not to build correctly.
```suggestion
```